### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by nikolaevtomov**
Note: Formatted the files after updating to the latest version of tslint.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,22 +34,22 @@
...
-    "prettier": "^1.16.4",
+    "@types/jest": "^24.0.22",
+    "jest": "^24.9.0",
+    "prettier": "^1.19.1",
     "request": "^2.88.0",
-    "ts-jest": "^24.0.2",
-    "tslint": "^5.11.0",
+    "ts-jest": "^24.1.0",
+    "tslint": "^5.20.1",
     "tslint-config-dabapps": "github:dabapps/tslint-config-dabapps#v0.5.3",
```
Comment (JakeSidSmith) [inline package.json]: Install from npm.

File: .travis.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ sudo: false
 language: node_js
 
 node_js:
-  - 6.11.5
+  - 10.5.0
```
Comment (JakeSidSmith) [inline .travis.yml]: Please remove this completely, Travis uses the nvmrc.

File: package.json [hunk 3]
```diff
@@ -34,23 +34,23 @@
   },
   "homepage": "https://github.com/dabapps/redux-requests#readme",
   "dependencies": {
-    "@types/js-cookie": "^2.0.28",
-    "@types/node": "^7.0.18",
-    "axios": "^0.18.0",
+    "@types/js-cookie": "^2.2.4",
+    "@types/node": "^12.12.7",
```
Comment (JakeSidSmith) [inline package.json]: Should match nvmrc.

Comment (nikolaevtomov) [reply package.json]: ta done

**Comment by nikolaevtomov** [comment]
Closing due to the special character in the branch name.

**Pull request closed without merging**